apps/src/templates/census2017/CensusInaccuracyReview.jsx

```diff
@@ -0,0 +1,400 @@
+import React, { PropTypes, Component } from 'react';
+import {Table} from 'reactabular';
+import Button from '../Button';
+
+export default class CensusInaccuracyReview extends Component {
+  static propTypes = {
+    authenticityToken: PropTypes.string,
+    reportsToReview: PropTypes.array,
+  };
+
+  state = {
+    toReview: undefined,
+    notes: '',
+    resolvedReports: [],
+  };
+
+  formatTeachesCs = (teachesCs) => {
+    return (
+      <div style={{textAlign: 'center', verticalAlign: 'middle'}}>
+        {teachesCs ? teachesCs : "Null"}
```

Comment:
We want the word "Null" rather than blank?

Reply:
It turns out this can't ever be null anyhow. I changed it to just: 
`{teachesCs}`